src/reducers/comments.spec.tsx

```diff
@@ -19,35 +25,80 @@ describe(__filename, () => {
 
   describe('beginComment', () => {
     it('begins a comment by key', () => {
-      const fileName = 'manifest.json';
-      const line = 123;
-      const versionId = 1;
+      const params = { versionId: 1, fileName: 'manifest.json', line: 123 };
```

Comment:
Nit: You could hoist the definition of `params` into the `describe('beginComment'` block and then not have to repeat them in these two tests. I think this would be okay because you never actually care about the specific values in `params` are really just referencing the `params` object in the tests. You might even be able to hoist them up to the top `describe(__filename` block, as it seems the same is true for most of the tests.